csatar-plugins/csatar/updates/builder_table_update_csatar_csatar_patrols.php

```diff
@@ -0,0 +1,65 @@
+<?php namespace Csatar\Csatar\Updates;
+
+use Schema;
+use October\Rain\Database\Updates\Migration;
+
+class BuilderTableUpdateCsatarCsatarPatrols extends Migration
+{
+    public function up()
+    {
+        Schema::table('csatar_csatar_scouts', function($table)
+        {
+            $table->integer('legal_relationship_id')->index('legal_relationship_id')->change();
+            $table->integer('special_diet_id')->index('special_diet_id')->change();
+            $table->integer('religion_id')->index('religion_id')->change();
+            $table->integer('tshirt_size_id')->index('tshirt_size_id')->change();
```

Comment:
```suggestion
            $table->integer('tshirt_size_id')->index('tshirt_size_id')->change();
            
            $table->foreign('user_id', 'user_foreign')
                ->references('id')
                ->on('users');
```
Ide még tegyük be ezt, mert ahogy én betettem nem működik.